src/storage/local_storage.cpp

```diff
@@ -97,7 +97,10 @@ idx_t LocalTableStorage::EstimatedSize() {
 	// get the index size
 	idx_t index_sizes = 0;
 	indexes.Scan([&](Index &index) {
-		index_sizes += index.GetInMemorySize();
+		// We expect that unbound indexes havent been instantiated yet and thus occupy no memory
+		if (index.IsBound()) {
+			index_sizes += index.Cast<BoundIndex>().GetInMemorySize();
```

Comment:
This should probably be an assertion - we shouldn't have unbound indexes in the transaction-local storage